### PR TITLE
test: Add tests to Create Event Module

### LINKED ISSUE #643: Update Tests for Create Event Module
**Is it a ?**

- [ ] Bug Report

- [ ] Feature Request
    
- [x] Chore
    

**Description**

The tests for Event Create Module does not cover the whole module. These should be updated.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by leeyh20**
Fixes #643.

Changes: Added tests to CreateEventPresenterTest and slightly modified current ones.

@iamareebjamal Are there any more tests that I should add?

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/app/module/event/create/CreateEventPresenter.java [hunk 1]
```diff
@@ -25,6 +25,9 @@
 
 public class CreateEventPresenter extends BasePresenter<ICreateEventView> implements ICreateEventPresenter {
 
+    public static String ERROR_DATE_END_AFTER_START_MESSAGE = "End time should be after start time";
+    public static String ERROR_DATE_WRONG_FORMAT_MESSAGE = "Please enter date in correct format";
+    public static String SUCCESS_MESSAGE = "Event Created Successfully";
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/module/event/create/CreateEventPresenter.java]: Don't store in contants as we'll be moving them to strings.xml in future. If you want to reference them in tests, just do it manually for now

Comment (leeyh20) [reply app/src/main/java/org/fossasia/openevent/app/module/event/create/CreateEventPresenter.java]: Okay!

**Comment by leeyh20** [comment]
@iamareebjamal Oh is there a reason why Codacy is unable to analyse this pull request?

**Comment by iamareebjamal** [comment]
No, it happens sometimes. Not an issue

**Pull request merged**